diff --git a/ext/extconf.rb b/ext/extconf.rb
index c21cdf1..e5d8ead 100644
--- a/ext/extconf.rb
+++ b/ext/extconf.rb
@@ -175,7 +175,7 @@ end
 # Platform-specific time functions
 if have_func('clock_gettime')
   # clock_gettime is POSIX, but the monotonic clocks are not
-  have_const('CLOCK_MONOTONIC_RAW', 'time.h') # Linux
+  #have_const('CLOCK_MONOTONIC_RAW', 'time.h') # Linux
   have_const('CLOCK_MONOTONIC', 'time.h') # Linux, Solaris, BSDs
 else
   have_func('gethrtime') # Older Solaris and HP-UX
